### PR TITLE
Fix nightly 04/05/2022

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix nightly 04/05/2022
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

**Review by SD1982 — DISMISSED** [review]

File: tests/UI/pages/BO/orders/view/tabListBlock.js [hunk 1]
```diff
@@ -432,12 +433,13 @@ class TabListBlock extends ViewOrderBasePage.constructor {
...
    * @param page {Page} Browser tab
-   * @param shippingData {{carrier: string, shippingCost: string, trackingNumber: string}} Data to set on shipping form
+   * @param shippingData {{trackingNumber: string, carrier: string, carrierID: number}} Data to set on shipping form
    * @returns {Promise<string>}
    */
   async setShippingDetails(page, shippingData) {
     await this.setValue(page, this.trackingNumberInput, shippingData.trackingNumber);
-    await this.setValue(page, this.carrierSelect, shippingData.carrier);
+    await page.click(this.carrierSelect);
+    await this.waitForSelectorAndClick(page, `${this.carriersSelectResult} li:nth-child(${shippingData.carrierID})`);
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/orders/view/tabListBlock.js]: Could you make the menu item a selector please ?

**Review by boubkerbribri — COMMENTED** [review]
One little comment and it's all good.

File: tests/UI/pages/BO/orders/view/tabListBlock.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view/tabListBlock.js]: Done thx :+1:

**Review by boubkerbribri — APPROVED** [review]
LGTM

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**